content/docs/get-started/index.md

```diff
@@ -165,3 +162,363 @@ requirements:
+allows us to easily decouple model training code from testing and deployment
+code.
+
+Let's try it out:
+
+<toggle>
+<tab title="Python Script">
+
+### Python code usage
+
+Load the model we saved earlier in a simple Python script to predict some
+probabilities.
+
+Create this `predict.py` script:
+
+```py
+from mlem.api import load
+
+model = load("models/rf")  # RandomForestClassifier
+features = [
+    "sepal length (cm)",
+    "sepal width (cm)",
+    "petal length (cm)",
+    "petal width (cm)",
+]
+x = pd.DataFrame([[0, 1, 2, 3]], columns=features)
+y_pred = model.predict_proba(x)
+
+print(y_pred)
+```
+
+Now, let's run the script:
+
+```cli
+$ python predict.py
+[[0.47 0.24 0.29]]
+```
+
+We see that the prediction probabilities were successfully printed to stdout.
+
+</tab>
+
+<tab title="Command Line">
+
+### Batch scoring
+
+The MLEM CLI allows us to natively use any saved model directly for prediction
+or batch scoring with any local dataset. This is very handy if we want to get
+some quick feedback about a model we just created.
+
+First, create an example dataset file to apply, we'll go with a `csv` format:
+
+```cli
+$ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3" > new_data.csv
+```
+
+Next, simply run `mlem apply` to apply this dataset against our model's
+`predict_proba` method:
+
+```cli
+$ mlem apply models/rf new_data.csv \
+    --method predict_proba \
+    --import \
+    --import-type "pandas[csv]"
+⏳️ Importing object from new_data.csv
+⏳️ Loading model from models/rf.mlem
+🍏 Applying `predict_proba` method...
+[[0.47 0.24 0.29]]
+```
+
+And we get our expected prediction probabilities as output.
+
+<details>
+
+#### Learn more about the CLI options used
+
+- The `--method`/`-m` flag tells MLEM to invoke the `predict_proba` method and
+  return the class probabilities, instead of the default `predict`.
+- The `--import`/`-i` flag tells MLEM to import the data on the fly.
+- The `--import-type` / `--it` flag, helps MLEM understand the data format.
+  Here, it's `pandas[csv]` - a csv file that should be read with Pandas. For
+  that to work, your data should be in a format that is supported by
+  [MLEM import](/doc/user-guide/importing). You can learn more about specifying
+  these arguments on `mlem apply` page.
+
+Alternatively, you could save the dataset itself
+[using MLEM](/doc/user-guide/data) to use `mlem apply` on it.
+
+</details>
+
+</tab>
+</toggle>
+
+We saw that MLEM provides a consistent and friendly way for you to work with
+models both via API and CLI. However, MLEM **really** shines when you need to
+package and deploy your models, either as part of an application or locally for
+testing.
+
+## Deploying and Serving models
+
+MLEM can serve a model for you using different server implementations, for
+example FastAPI or RabbitMQ. Here we'll check out how it works with FastAPI
+since serving models via a REST API is a very common use case.
+
+## Local model serving
```

Comment:
Confusing structure...

Reply:
Not sure I get what you mean here exactly - I totally see a room for confusion between serving and deploying. I think it might be a real confusion in MLEM's UX we need to think how to improve.
Is that what you mean? or something else? can you elaborate a bit?

Reply:
I've started addressing this in a follow-up PR. Here's some more context: https://github.com/iterative/mlem.ai/pull/265#discussion_r1047651762

> room for confusion between serving and deploying

Exactly. I'm not sure yet if it's something docs-specific or more of a product question (UX) indeed. But surely we can at least ameliorate the issue with good docs structure.

Reply:
Wait no, the link should be: https://github.com/iterative/mlem.ai/pull/265#pullrequestreview-1216690806

Reply:
I think it's a real product UX question - cc @aguschin @mike0sv let's discuss this a bit, brainstorm if we can simplify or communicate this differently - I get the confusion @jorgeorpinel has here 💯 .

For now I've built this and the next [proposed iteration ](https://github.com/iterative/mlem.ai/pull/281) of get-started taking the current abstraction/structure as given (deployment builds on top serving) since the actual code is structured around this concept

Reply:
Nice, thanks. We may have conflicts among PRs though... I'll focus on finishing mine first.